src/screepsServer.ts

```diff
@@ -46,32 +46,43 @@ export class ScreepsServer extends EventEmitter {
         this.lastAccessibleRoomsUpdate = -20;
```

Comment:
Right now, this member variable is never read or changed after this point. Should it be removed?

Reply:
Resolving for now, I'll put up another small PR to remove these options, and we can accept or deny it there instead of holding this up.